src/components/LiveChatLoaderProvider.tsx

```diff
@@ -5,11 +5,12 @@ import { LiveChatLoaderContext } from '../context'
 
 interface LiveChatLoaderProps {
   provider: Provider
-  children: JSX.Element
+  children: React.ReactNode
   idlePeriod?: number
   providerKey: string
   appID?: string
   baseUrl?: string
+  locale?: string
```

Comment:
Is `locale` is used by all implementations?

Reply:
Only **chatwoot** and **messenger** currently use `locale` - but in both instances, it's required during initialisation, hence why it's been implemented at the global `LiveChatLoaderProvider` level

Reply:
Seems worthy of a comment

Reply:
Good call. Done ✅